lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+const Formatter = require('../../formatters/formatter');
+
+class ExternalAnchorsUseRelNoopenerAudit extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Performance',
+      name: 'external-anchors-use-rel-noopener',
+      description: 'Site opens external anchors using rel="noopener".',
+      helpText: 'links that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener">opening page\'s performance does not suffer.</a>',
+      requiredArtifacts: ['ExternalAnchorsWithNoRelNoopener']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    if (typeof artifacts.ExternalAnchorsWithNoRelNoopener === 'undefined' ||
+        artifacts.ExternalAnchorsWithNoRelNoopener === -1) {
+      return ExternalAnchorsUseRelNoopenerAudit.generateAuditResult({
+        rawValue: -1,
+        debugString: 'ExternalAnchorsWithNoRelNoopener gatherer did not run'
+      });
+    }
+
+    const failingAnchors = artifacts.ExternalAnchorsWithNoRelNoopener.usages.map(anchorHref => {
+      return {
+        url: anchorHref
```

Comment:
This will just show anchors' URLs  when presented by the URL formatter. Not sure how helpful that will be. How about re-constructing the author's `<a>` snippet. If you had the actual `HTMLAnchorElement`s from the gatherer instead of just URLs strings, you could do:

url: \`${anchor.outerHTML}\`